torch/_decomp/__init__.py

```diff
@@ -58,6 +60,10 @@ def _add_op_to_registry(registry, op, fn):
         # to filter those out, e.g aten.add.float_int
         if torch._C._dispatch_has_kernel(op_overload.name()):
             registry[op_overload] = fn
+        else:
+            log.warning(
```

Comment:
Do we want to warn here at all?

Reply:
When `overloads_with_kernels()` is used instead of `overloads()`, this warning is never realized. I found this warning useful while debugging pre-overloads_with_kernels code but I am fine with removing it now if you insist :)